backend/spec/services/translations/translate_project_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe Translations::TranslateProject do
+  let(:project) {
+    translatable_columns = Translations::TranslateProject.translatable_columns
+    attrs = translatable_columns.map { |col| ["#{col}_en", "test"] }.to_h
+    create(:project, attrs.merge(name_en: "Project name")) # for simplicity just one translatable attribute present
```

Comment:
Yup, I think it is fine :), I don't think we need to run translation for every translatable column and every language